prototype/osl-readme.md

```diff
@@ -0,0 +1,107 @@
+# Creating an `asf_tools` prototype environment in OpenSARlab
+
+<https://opensarlab.asf.alaska.edu/user/>
```

Comment:
this gives "page not found" as written, may need to be `/user/<username>`

Reply:
oh, right! Think I'll drop `/user/`